client/src/main/scala/io/delta/sharing/client/filters/JsonPredicates.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.delta.sharing.spark.filters
+package io.delta.sharing.client.filters
```

Comment:
Why do we need to move the predicates?
The client treats them as a json string right?

Reply:
good point. moved back to spark.